src/Components/Notes/NoteCard.jsx

```diff
@@ -169,16 +165,26 @@ export default function NoteCard({
     setComments(newComments)
   } */
 
-
   /** Update issue on GH, set read-only */
-  async function submitUpdate() {
+  async function updateIssueGithub() {
     const res = await updateIssue(repository, noteNumber, title, editBody, accessToken)
     const editedNote = notes.find((note) => note.id === id)
     editedNote.body = res.data.body
```

Comment:
Fix this?  https://github.com/bldrs-ai/Share/issues/1201

Reply:
this code is related to the issues, i will do it in the separate PR.
Will keep this PR on comments